src/frontend/src/components/CreateGallery.tsx

```diff
@@ -0,0 +1,133 @@
+import React from 'react';
+import Image from "next/image";
+// import GalleryItem from './GalleryItem';
+import { IoIosArrowBack, IoIosArrowForward, IoIosAdd } from "react-icons/io";
+import { HiOutlinePlusSm } from "react-icons/hi";
+import { GrLocation } from "react-icons/gr";
+
+const CreateGallery = () => {
+  return (
+    <div className={`mx-auto mt-20 rounded-lg shadow-[rgba(50,50,93,0.25)_0px_6px_12px_-2px,_rgba(0,0,0,0.3)_0px_3px_7px_-3px]`} style={{ maxWidth: '70vw', height: '72vh', backgroundColor: 'white', display: 'flex', alignItems: 'center', justifyContent: 'space-between'}}>
+      <div className='left-section'>
+        <div className="top-left-section flex items-center ml-12">
+          {/* <IoIosArrowBack className="text-black mr-5" style={{ fontSize: '3rem' }} /> */}
+          <button className="text-black mr-5" style={{ fontSize: '3rem'}}>
+            <IoIosArrowBack />
+          </button>
+          
+          <div className="flex flex-col items-center justify-center rounded-lg" style={{ width: '25vw', height: '30vh', backgroundColor: '#FBFAFA' }}>
+            <Image src={"/images/UploadToCloud.png"}
+                    alt="picKeeper"
+                    width={100}
+                    height={100}
+                    />
+            {/* <input type="file" id="fileInput" className="hidden" /> */}
+            <button className="mt-8 text-white rounded px-6 py-1" style={{ backgroundColor: '#E19007'}}>Browse</button>
+            <div className="mt-4">
+              <span className="text-gray-500">Max file size : 20 MB</span>
+            </div>
+            <div className="mt-1">
+              <span className="text-gray-500">Supported file types : JPG, PNG</span>
+            </div>
+          </div>
+          
+          <button className="text-black ml-5" style={{ fontSize: '3rem'}}>
+            <IoIosArrowForward/>
+          </button>
+        </div>
+        <div className="down-left-section items-start mt-20 ml-10">
+          <div className="flex items-center justify-center rounded-2xl hover:border-yellow-500 hover:border-4 hover:shadow-[0px_0px_0px_1px_rgba(0,0,0,0.06),0px_1px_1px_-0.5px_rgba(0,0,0,0.06),0px_3px_3px_-1.5px_rgba(0,0,0,0.06),_0px_6px_6px_-3px_rgba(0,0,0,0.06),0px_12px_12px_-6px_rgba(0,0,0,0.06),0px_24px_24px_-12px_rgba(0,0,0,0.06)]" style={{ width: '7vw', height: '18vh', backgroundColor: '#FBFAFA', position: 'relative' }}>
```

Comment:
use shadow-... tailwind class for consistency and add the style in className instead